static/script-tests/tests/devices/mediaplayer/cehtmlcommontests.js

```diff
@@ -1369,6 +1369,46 @@ window.commonTests.mediaPlayer.cehtml.mixinTests = function (testCase, mediaPlay
         });
     };
 
+    mixins.testGetDurationReturnsInfinityWithALiveVideoStream = function(queue) {
```

Comment:
We didnt move the tests to common tests because the way the mocking was done seemed to be different in html5. But on looking at it again, it seems that the mocking can be made similar in all the cases by using deviceMockingHooks. Thanks for the suggestion. Will move the tests to common tests.